cli/root.go

```diff
@@ -93,7 +95,19 @@ func NewAlertmanagerClient(amURL *url.URL) *client.Alertmanager {
 		cr.DefaultAuthentication = clientruntime.BasicAuth(amURL.User.Username(), password)
 	}
 
-	return client.New(cr, strfmt.Default)
+	c := client.New(cr, strfmt.Default)
+
```

Comment:
Given this is now executed for _every_ command, can we an option to skip it? 

Something like:

```golang
var skipVersionCheck bool

app.Flag("skip.version-check", "Skip version checks when executing commands.").BoolVar(&skipVersionCheck)

// Then, we can do something like
if skipVersionCheck {
	return c
}

```